crates/lib/src/games/definitions.rs

```diff
@@ -124,7 +124,7 @@ pub static GAMES: Map<&'static str, Game> = phf_map! {
     "soulmask" => game!("Soulmask", 27015, Protocol::Valve(Engine::new(2_646_460))),
     "serioussam" => game!("Serious Sam", 25601, Protocol::Gamespy(GameSpyVersion::One)),
     "squad" => game!("Squad", 27165, Protocol::Valve(Engine::new(393_380))),
-    "theforest" => game!("The Forest", 27016, Protocol::Valve(Engine::new(556_450))),
+    "theforest" => game!("The Forest", 27016, Protocol::Valve(Engine::new(242_760))),
```

Comment:
We can include both ids here so that we support both types of host

```
"theforest" => game!("The Forest", 27016, Protocol::Valve(Engine::new_with_dedicated(242_760, 556_450))),
```

This should fix the issue

Reply:
Oh, nice! I wondered if there was a better way, should have poked around a bit more. Will get that added now, thanks!